webapp/src/app.tsx

```diff
@@ -781,7 +781,8 @@ export class ProjectView
             orphanException: brk => {
                 // TODO: start debugging session
                 // TODO: user friendly error message
-                core.warningNotification(lf("Program Error: {0}", brk.exceptionMessage));
+                //core.warningNotification(lf("Program Error: {0}", brk.exceptionMessage));
+                this.textEditor.onExceptionDetected(brk)
```

Comment:
don't assume the user is in the textEditor, use currentEditor.

Reply:
since the errorlist is currently only available in the text editor would it be safe to only call this if its the current active editor and use the previous call for the other editors for the time being?

Reply:
This should be set on all editors and have a base function that does nothing